api/config/cdr_versions_prod.json

```diff
@@ -19,7 +19,7 @@
     "creationTime": "2018-11-13 15:09:27Z",
     "releaseNumber": 1,
     "numParticipants": 116157,
-    "cdrDbName": "r_2019q1_2"
+    "cdrDbName": "r_2019q1_3"
```

Comment:
Is the new CDR `r_2019q2_1` already on the latest schema?

Reply:
I had to generate another version from R2018Q4R1 that includes the new criteria table. The prior version r_2019q1_2 only had the old criteria table and r_2019q2_1 was generated from R2019Q2R1.

Reply:
Makes sense - my question was just to double-check whether the **other** CDR already has the new tables (All of Us Dataset v2 below).

Reply:
Ah.. sorry.. misunderstood your question. Yes that is correct that r_2019q2_1 has the new criteria tables.